sdc/tests/tests_perf/test_perf_df.py

```diff
@@ -198,15 +127,5 @@ def {func_name}(self):
 ]
 
 
-def gen(cases, method):
-    for params in cases:
-        func, param, length = params
-        name = func
-        if param:
-            name += "_" + to_varname(param).replace('__', '_')
-        func_name = 'test_df_{}'.format(name)
-        setattr(TestDataFrameMethods, func_name, method(func, param, length))
-
-
-gen(cases, test_gen)
-gen(cases_two_par, test_gen_two_par)
+generate_test_cases(cases, TestDataFrameMethods, 'df')
```

Comment:
Generators and test data specified for tests should be a part of corresponding tests.
The same in another places.